.gitignore

```diff
@@ -43,7 +43,3 @@ out/
 build/
 gradle-build/
 .gradle/
-
-# nodejs
-node_modules/
-package-lock.json
```

Comment:
Why is this necessary?

Reply:
it was automatically added. But we don't add node_modules and package-lock since we don't have any front-end components in this project.